improver/metadata/amend.py

```diff
@@ -66,6 +68,22 @@ def update_stage_v110_metadata(cube: Cube) -> None:
     cube.attributes["mosg__grid_version"] = "1.1.0"
 
 
+def _load_json(json_path: str) -> Dict:
+    """Load a JSON file into a dictionary.
+
+    Args:
+        json_path:
+            Path to the JSON file to load.
+
+    Returns:
+        Dictionary containing the JSON data.
+    """
+    if not isinstance(json_path, str):
+        return json_path
+    json_path = os.path.expanduser(os.path.expandvars(json_path))
+    return json.load(open(json_path, "r"))
```

Comment:
Is the intention to reuse this loader in other instances of json loading? If so, it might be better for it to live in https://github.com/metoppv/improver/blob/master/improver/utilities/load.py despite it being in this case related to the `amend_attributes` function.

Reply:
Similar to Ben's comment, I would also expect this function to be located centrally with the other loading functionality.

Reply:
Yes. Further to this, I think the JSON loader should be a Plugin like the NetCDF loader, so that the workflow graph can have a node to load JSON data as a `dict` object.